### PR TITLE
Add stats: like meme section to Developer Guide

### PR DESCRIPTION
**Pull request by moziliar**

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -356,6 +356,56 @@ For example, `a` to add a piece of text, arrow keys to adjust the position of th
 
 // end::templates[]
 
+// tag::stats[]
+=== Statistics feature 1: Like Meme
+
+Unlike other file management system, people have preference of some memes over others.
```
Comment (jonchan51) [inline docs/DeveloperGuide.adoc]: The part on file management system doesnt really make sense

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -356,6 +356,56 @@ For example, `a` to add a piece of text, arrow keys to adjust the position of th
...
+
+The following sequence diagram shows how `MemeLikeCommand` communicates with `Stats` and update the like count.
+
+image::LikeMemeSequenceDiagram.png[]
+
+==== Design Considerations
+
+===== Aspect: Implementation of LikeData.
+
+* **Alternative 1:** put like data as a field inside Meme object.
```
Comment (jonchan51) [inline docs/DeveloperGuide.adoc]: Can reformat all of these so the first letter is a capital letter?

File: docs/DeveloperGuide.adoc (hunk 1 above)
Comment (moziliar) [reply docs/DeveloperGuide.adoc]: Huh? Ppl don't just mark some files "Prefered" over others i guess. It is different from tag as well.

File: docs/DeveloperGuide.adoc (hunk 2 above)
Comment (moziliar) [reply docs/DeveloperGuide.adoc]: Oh ok

**Pull request merged**